docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1d.md

```diff
@@ -0,0 +1,82 @@
+           -  add the airflow to the zone total exhaust: `zone_total_exhaust += exhaust_fan.design_airflow`
+
+       - Get a list of HVAC systems serving the zone: `hvac_sys_list_serving_zone_x =  get_list_hvac_systems_associated_with_zone(RMI,zone.id)`  
+       - For each hvac system serving the zone: `for hvac in hvac_sys_list_serving_zone_x:`  
+           - Check if there is a fan system associated with the hvac system (if not, then there is no central HVAC system exhaust fan (terminal units don't have exhaust fans)): `if hvac.fan_system != Null:`  
+               - Get list of zones served by the hvac system: `zone_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["ZONE_LIST"]`  
+               - create a variable to hold the total exhaust airflow rate of the HVAC system: `hvac_system_total_exhaust_airflow = 0`
+               - Look at each fan in the fan_system.exhaust_fans: `for exhaust_fan in fan_system.exhaust_fans:`
+                   - add the exhaust fan airflow to the hvac exhaust airflow total: `hvac_system_total_exhaust_airflow += exhaust_fan.design_airflow`
+               - Check if the hvac system serves more than one zone, if it does then carry on, if not then go to Else: `if len(zone_list_hvac_sys_x) >1:`  
+                   - Get list of terminal units served by the hvac system: `terminal_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["Terminal_Unit_List"]`  
+                   - create a variable for the total terminal air flow for the hvac system - this total terminal airflow is used to divide exhaust proportionally across zones: `total_terminal_air_flow = 0`
+                   - create a variable to hold the terminal airflow for this zone: `zone_primary_air_flow = 0`
+                   - For each terminal unit served by the HVAC system: `for terminal in terminal_list_hvac_sys_x:`  
+                       - Get the primary air flow: `primary_air_flow_terminal = terminal.primary_airflow`  
+                       - Add to the total cfm across all terminals served by the HVAC system: `total_terminal_air_flow = total_terminal_air_flow + primary_air_flow_terminal`  
+                       - Check if the terminal unit is associated with this zone: `if terminal in zone.terminals:`   
+                           - Add to the primary flow for the zone for this hvac system:  `zone_primary_air_flow = zone_primary_air_flow + primary_air_flow_terminal`  
+                   - Get the HVAC system's FanSystem: `fan_system = hvac.fan_system`
+                   - Apportion the exhaust airflow to the zone based on the terminal unit primary flow for the zone if hvac_system_total_exhaust_airflow > 0 and zone_primary_air_flow > 0: `if( zone_primary_air_flow > 0 && hvac_system_total_exhaust_airflow > 0 ): zone_total_exhaust += hvac_system_total_exhaust_airflow / zone_primary_air_flow`
```

Comment:
I am not quite following this logic. Should it be zone_total_exhaust += hvac_system_total_exhaust_airflow * zone_primary_air_flow / total_terminal_air_flow

Reply:
right.  changed